internal/ogc/tiles/main_test.go

```diff
@@ -233,13 +233,13 @@ func TestTiles_Tile(t *testing.T) {
 			fields: fields{
 				configFile:      "internal/ogc/tiles/testdata/config_tiles_urltemplate.yaml",
```

Comment:
Wat extra testcases waarin we de error handling raken lijkt mij goed. De ogc conformance test zie ik vooral als extra.